packages/grid/vpn/headscale.py

```diff
@@ -16,6 +17,11 @@
 
 network_name = os.environ.get("NETWORK_NAME", "omnet")  # default to omnet
 
+key = os.environ.get("STACK_API_KEY", None)  # Get key from environment
```

Comment:
If we have no environment variable set lets just exit that way we cant accidentally be missing one.